lib/components/component.dart

```diff
@@ -134,3 +135,26 @@ class SpriteComponent extends PositionComponent {
   @override
   void update(double t) {}
 }
+
+class SvgComponent extends PositionComponent {
```

Comment:
Create this class on its own file, please! This file is already too big.

Reply:
That was my initial aproach, but then I realized that the basic components are all here and I wanted to keep the same behaviour. Would it not be better to refactor this after and create a file for each component?

Reply:
I think it would be better to have each on its own file, but we can keep it like so for now, no problem.